bytes_test.go

```diff
@@ -65,7 +65,7 @@ func TestByteErrors(t *testing.T) {
 	if err == nil {
 		t.Errorf("Expected error, got %v", got)
 	}
-	got, err = ParseBytes("")
+	_, err = ParseBytes("")
```

Comment:
If we're not expecting a value here, it'd be good to report what we got that we didn't expect.